src/get-config-for-method.js

```diff
@@ -96,9 +96,27 @@ export default ({
   }
 
   if (hasInclude) {
+    const modelsHasAssociations = modelAssociations && modelAssociations.length;
+    const validAssociationsString = modelsHasAssociations
+      ? joi.string().valid(...modelAssociations)
+      : joi.valid(null);
+    const validAssociationsObject = modelsHasAssociations
+      ? joi.object().keys({
+        model: joi.string().valid(...modelAssociations),
+        where: joi.object().keys({
+          ...attributeValidation,
+          ...sequelizeOperators,
+        }),
+      })
+      : joi.valid(null);
     const query = concatToJoiObject(joi.object()
       .keys({
-        ...associationValidation,
+        include: [
+          joi.array().items(validAssociationsString),
```

Comment:
hmmm… it's weird that we're adding in custom validation here instead of just trusting that the `associationValidation` is correct when we received it. We should fix this at the source in `crud.js`